app/routes/create-catch-certificate/$catchCertificate/what-are-you-exporting.tsx

```diff
@@ -20,40 +19,63 @@ export const loader: LoaderFunction = async ({ params, request }: DataFunctionAr
 };
 
 export const action: ActionFunction = async ({ request, params }): Promise<Response> => {
-  const { catchCertificate } = params;
-  const _redirect = `/create-catch-certificate/${catchCertificate}/what-are-you-exporting`
   const form = await request.formData();
-  const { _action, ...values } = Object.fromEntries(form);
-
+  const { catchCertificate } = params;
+  const { _action, faoName, ...values } = Object.fromEntries(form);
+  const _redirect = `/create-catch-certificate/${catchCertificate}/what-are-you-exporting`;
+  
   if (_action === 'cancel') {
     return redirect(`${_redirect}#add-products`);
   }
 
-  const resquestBody: any = {
-    addToFavourites: false,
-    btn_submit:"",
-    presentationLabel:"",
+  if (_action === 'addSpecies') {
```

Comment:
is it a switch case?

Reply:
Updated to use switch case @daniele-zurico